xml/System.Text/StringRuneEnumerator.xml

```diff
@@ -124,9 +129,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns the current enumerator instance.</summary>
+        <returns>The current enumerator instance.</returns>
```

Comment:
We don't remove the remarks. It might come in handy if people need to add them in the future.

```suggestion
        <returns>The current enumerator instance.</returns>
        <remarks>To be added.</remarks>
```